TIL/daily/2025-03-21.md

```
@@ -0,0 +1,245 @@
+- person1과 person2는 동일하지 않지만
+- 두 객체의 address는 동일하다
+
+### 깊은 복사
+
+실제 값을 새로운 메모리 공간에 복사하는 것
+
+- 새로운 메모리 공간에 복사되었기 때문에 하나를 변경해도 같이 변경되지 않는다
+
+```dart
+class Address {
+	String street;
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)